float32 droll
float32 dpitch
float32 dyaw
